dm/master/server.go

```diff
@@ -1739,22 +1739,28 @@ func extractWorkerError(result *pb.ProcessResult) error {
 
 // waitOperationOk calls QueryStatus internally to implement a declarative API. It will determine operation is OK by
 // Source:
-//   OperateSource:
-//     * StartSource, UpdateSource: sourceID = Source
-//     * StopSource: return resp.Result = false && resp.Msg = “worker has not started”.
+//
+//	OperateSource:
+//	  * StartSource, UpdateSource: sourceID = Source
+//	  * StopSource: return resp.Result = false && resp.Msg = “worker has not started”.
+//
 // Task:
-//   StartTask, UpdateTask: query status and related subTask stage is running
-//   OperateTask:
-//     * pause: related task status is paused
-//     * resume: related task status is running
-//     * stop: related task can't be found in worker's result
+//
+//	StartTask, UpdateTask: query status and related subTask stage is running
+//	OperateTask:
+//	  * pause: related task status is paused
+//	  * resume: related task status is running
+//	  * stop: related task can't be found in worker's result
+//
 // Relay:
-//   OperateRelay:
-//     * start: related relay status is running
-//     * stop: related relay status can't be found in worker's result
-//   OperateWorkerRelay:
-//     * pause: related relay status is paused
-//     * resume: related relay status is running
+//
+//	OperateRelay:
+//	  * start: related relay status is running
+//	  * stop: related relay status can't be found in worker's result
+//	OperateWorkerRelay:
+//	  * pause: related relay status is paused
+//	  * resume: related relay status is running
+//
```

Comment:
seems markdown is not recgonized, I prefer

```
// Source:
//
//	OperateSource:
//	  - StartSource, UpdateSource: sourceID = Source
//	  - StopSource: return resp.Result = false && resp.Msg = “worker has not started”.
//
// Task:
//
//	StartTask, UpdateTask: query status and related subTask stage is running
//
//	OperateTask:
//	  - pause: related task status is paused
//	  - resume: related task status is running
//	  - stop: related task can't be found in worker's result
//
// Relay:
//
//	OperateRelay:
//	  - start: related relay status is running
//	  - stop: related relay status can't be found in worker's result
//
//	OperateWorkerRelay:
//	  - pause: related relay status is paused
//	  - resume: related relay status is running
//
```

Reply:
I change the code to make it close to your suggestion.